src/app/utils/field-validation/validators/countryRegionValidator.ts

```diff
@@ -0,0 +1,33 @@
+import { chain, left, right } from 'fp-ts/lib/Either'
+import { flow } from 'fp-ts/lib/function'
+
+import { CountryRegion } from '../../../../../shared/constants/countryRegion'
+import { ResponseValidator } from '../../../../types/field/utils/validation'
+import { ProcessedSingleAnswerResponse } from '../../../modules/submission/submission.types'
+
+import { notEmptySingleAnswerResponse } from './common'
+import { isOneOfOptions } from './options'
+
+type CountryRegionValidator = ResponseValidator<ProcessedSingleAnswerResponse>
+type CountryRegionValidatorConstructor = () => CountryRegionValidator
+
+/**
+ * Returns a validation function
+ * to check if country/region selection is one of the options.
+ */
+const makeCountryRegionValidator: CountryRegionValidatorConstructor =
+  () => (response) => {
+    const validOptions = Object.values(CountryRegion)
```

Comment:
Current validation is failing because the CountryRegion enum (used to display the dropdown options) are in titlecase - might need to map `.toUppercase()` here for the validation to pass

Reply:
I see..., I did not manage to catch this from the test error message... I submitted a fix, will see how it goes.